.github/PULL_REQUEST_TEMPLATE

```diff
@@ -29,7 +29,12 @@ Security Update?
 Build Order
 -----------
 
-Please describe in what order maintainers should build this pull request.
+<!-- Please describe in what order maintainers should build this pull request. You can use the following template, use space to separate packages. -->
+
+```
+PACKAGES...
```

Comment:
May be better denoted by `pkg1 pkg2 pkg3 ...`.

Reply:
Done